src/App.js

```diff
@@ -71,21 +74,18 @@ class App extends Component {
                 <li className="App-menu__item">
                   <NavLink to="/registration">Registration</NavLink>
                 </li>
-
               </ul>
             </header>
             <div>
               <Switch>
-                <Route exact path="/dashboard" component={Homepage} />
+                <Route exact path={["/dashboard", "/"]} component={Homepage} />
```

Comment:
co znaczy ta zmiana? nie do końca ją rozumiem

Reply:
Bez tej reguły klient wchodząc na adres główny zostałby przekierowany wg routingu na strone "page not found".
dzieki temu jeżeli w adresie bedzie "/" oraz "/dashborad" bedzie wczystany dashboard.